readded everything (#104)
MrLongbottom/P10-Article: sections/conclusion.tex

\section{Conclusion}\label{sec:conclusion}

> In this paper, we explore possibilities for incorporating metadata\vejleder{gerne ??} into existing topic models, such as \gls{lda} and \gls{pam}.

Reviewer: ```suggestion
In this paper, we have explored possibilities for incorporating metadata\vejleder{gerne ??} into existing topic models, such as \gls{lda} and \gls{pam}.
```

> We evaluate these models based on three different metadata: Author, Category, and Taxonomy, each of which represents the data in a different way.

Reviewer: ```suggestion
We evaluated these models based on three different metadata: Author, Category, and Taxonomy, each of which represents the data in a different way.
```

Reviewer 2: ```suggestion
We have constructed models based on three different types of metadata: Author, Category, and Taxonomy, each of which represents the data in a different way.
We evaluated our models based on topic coherence and found that the Taxonomy model was the best-performing model for our dataset.
```

> We evaluate these models based on three different metadata: Author, Category, and Taxonomy, each of which represents the data in a different way.

Reviewer 2: ```suggestion
We evaluate our models based on topic coherence and conducted an analysis that investigates these models.
```

> From the topic coherence results, shown in \autoref{tab:metric_results}, the \gls{pam} model using the Taxonomy metadata gets the best results.

Reviewer: ```suggestion
From the topic coherence results, shown in \autoref{tab:metric_results}, the \acrfull{pam} using the Taxonomy metadata gets the best results.
```

> The Author and Category models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models.\vejleder{uddyb gerne denne paragraf}

Reviewer 2: ```suggestion
The Author-topic and Category-topic models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models. 
This can be due to the authors within journalism cover a broader range of topics than within scientific literature, which can negatively impact the topic coherence.
```

Reviewer: ```suggestion
The Author-topic and Category-topic models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models. 
This can be due to the authors within journalism covering a broader range of topics than within scientific literature, which can negatively impact the topic coherence.
```
covering

> The Author and Category models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models.\vejleder{uddyb gerne denne paragraf}

Reviewer: ```suggestion
The Author-topic and Category-topic models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models.\vejleder{uddyb gerne denne paragraf}
```

> We want to answer the problem statement, which we stated in \autoref{sec:introduction}:

Reviewer: ```suggestion
We want to answer the research questions, which we stated in \autoref{sec:introduction}:
```

\vejleder{lidt rodet måske.}

> \begin{itemize}
> 	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
> 	\item \textit{What possible problems can these models help alleviate at Nordjyske?}
> 	\begin{itemize}
> 		\item \textit{More specifically, how can these models improve recommendation?}
> 	\end{itemize}
> \end{itemize}

Reviewer 2: ```suggestion
\begin{itemize}	
    \item \textit{How can we establish models that incorporate metadata from the Nordjyske dataset?}	
    \item \textit{How does including metadata within such models impact the resulting topics?}
 \end{itemize}
```

We incorporate metadata into our topic models in multiple ways.
The \gls{lda} model is used as a baseline, which indicates whether incorporating metadata can improve the topic quality of our topic models.

> Based on the results of Author-Topic and Category-Topic, we see that only using the metadata for topic assignment within \gls{lda} can hurt the topic quality.

Reviewer 2: ```suggestion
Based on the results of the Author-Topic and Category-Topic, we see that only using the metadata for word topic assignment within \gls{lda} can hurt the topic quality.

```

Author: andre steder har vi lille t i 'topic' i 'Author-topic'

Author: ```suggestion
Based on the results of the Author-topic and Category-topic models, we see that only using the metadata for word topic assignment within \gls{lda} can hurt the topic quality.
```

> Other studies have shown that only using the metadata for topic assignment can improve the performance.

Reviewer 2: ```suggestion
Other studies have shown that only using the metadata for word topic assignments can improve performance~\cite{MetaLDA2017} \cite{author_topic_2012}.
```

> This might be due to the particularities of the dataset, such as authors not usually writing about the same subject within the news environment.  

Reviewer: ```suggestion
This might be due to the particularities of our dataset, such as authors not usually writing about the same subject within the news environment.  
```

> 

Reviewer: stor spacing i artiklen?

> We use the \gls{pam} to incorporate a hierarchically structured metadata called Taxonomy, where we use a novel locking mechanism to lock the observed metadata's topics into place.

Reviewer: ```suggestion
We use the \gls{pam} to incorporate a hierarchically structured taxonomy metadata, where we use a novel locking mechanism to lock the observed metadata's topics into place.
```

> Using this model and technique, we can get better topic quality and words within \gls{pam} compared to the \gls{lda}.

Reviewer: ```suggestion
Using this model and technique, we can get better topic quality within \gls{pam} compared to the \gls{lda} model.
```

However, the run time of the algorithm is quite slow compared to the \gls{lda}.

> All models ran for 50 epochs, but \gls{pam} had a runtime of 130 hours, compared to \gls{lda} which had a runtime of 8 hours.

Reviewer 2: make sure this is the same across the whole article

> All models ran for 50 epochs, but \gls{pam} had a runtime of 130 hours, compared to \gls{lda} which had a runtime of 8 hours.

Reviewer: ```suggestion
All models ran for $50$ epochs, but \gls{pam} had a runtime of $132$ hours, compared to \gls{lda} which had a runtime of $8$ hours.
```
132 står der i Results

> Topic modeling can be used to alleviate many problems at Nordjyske, such as recommendations.

Reviewer: ```suggestion
Topic modeling can be used to support recommendation in different ways.
```

We can use the topic distributions from our models to compare articles based on their similarity in topics.

> For example, the author-topic model's topic distribution can be used to recommend similar authors, while in the topic distributions of the taxonomy model, there is the possibility of looking at topics from different layers.

Reviewer: ```suggestion
For example, the author-topic model's topic distribution can be used to recommend similar authors, while for the topic distributions of the taxonomy model, there is the possibility of looking at topics from different layers.
```

This information can be used in a content-based filtering approach to recommend similar articles.

> We suggest using this topic information to enrich the recommendation process at Nordjyske. 

Reviewer 2: i am not sure about this

> We suggest using this topic information to enrich the recommendation process at Nordjyske. 

Reviewer: ```suggestion
We suggest using this topic information to enrich the recommendation process at news sites. 

```
ny paragraf til future work

Author: ```suggestion
We suggest using this topic information to enrich the recommendation process at news sites, such as Nordjyske. 

```

Reviewer 2: ```suggestion

```
